src/services/SearchCarousel/SearchCarouselServices.types.ts

```diff
@@ -0,0 +1,16 @@
+type searchCarouselName = {
+  en_US: string;
+  ar_US: string;
+};
+
+export type SearchCarouselDataType = {
+  name: searchCarouselName;
+  image: string;
+  buttonText: string;
+  category: string;
+};
+
+export type SearchCarouselDataSuccessResponse = {
```

Comment:
Add TODO to change this to SuccessResponse < SearchCarouselDataType []>

Reply:
Added